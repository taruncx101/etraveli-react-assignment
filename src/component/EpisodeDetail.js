import { Row, Col } from 'react-bootstrap';
import { Fragment } from 'react';
function EpisodeDetail({ selectedEpisode }) {
    if (!selectedEpisode) {
        return (<Row>
            <Col>No Movie Selected</Col>
        </Row>)
    } else {
        return (
            <Fragment>
                <Row>
                    <Col>Episode {selectedEpisode?.roman_id} -  {selectedEpisode?.title}</Col>

                </Row>
                <Row>
                    <Col>{selectedEpisode?.opening_crawl}</Col>
                </Row>
            </Fragment>

        )
    }

}

export default EpisodeDetail;