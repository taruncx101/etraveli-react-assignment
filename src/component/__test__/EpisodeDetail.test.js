import renderer from 'react-test-renderer';
import EpisodeDetail from '../EpisodeDetail';


it('render without crashing', () => {
  const component = renderer.create(
    <EpisodeDetail selectedEpisode={null}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
