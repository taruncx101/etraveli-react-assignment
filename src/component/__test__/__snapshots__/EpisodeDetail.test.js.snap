// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crashing 1`] = `
<div
  className="row"
>
  <div
    className="col"
  >
    No Movie Selected
  </div>
</div>
`;
